blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/OrganizationImpl.java

```diff
@@ -8,19 +8,21 @@
 import io.jenkins.blueocean.commons.stapler.JsonBody;
```

Comment:
nit: I noticed that there is some unused code around `update` - could that be cleaned up?